update_multiarch.sh

```diff
@@ -71,6 +71,8 @@ do
 				if [ "${vm}" != "hotspot" ]; then
 					reldir="${reldir}-${vm}";
 				fi
+
+				echo "${file} ${build} ${btype} ${os}"
```

Comment:
Is this intentional or is this a debug statement that was left out.

Reply:
left from debugging, sorry